common/config.js

```diff
@@ -55,7 +55,7 @@ module.exports = {
       },
     },
     sentencePlanning: {
-      url: get('SENTENCEPLANNINGAPI_ENDPOINT_URL', 'http://localhost:8080', true),
+      url: get('SENTENCEPLANNINGAPI_ENDPOINT_URL', 'http://localhost:18081', true),
```

Comment:
I commit this every time too 🤣 
Firstly, get rid.
Secondly, we need to come up with a way of auto switching the port depending on if you're running a local env.